src/Application/Filter/AdmissionRequirementTypeFilter.php

```diff
@@ -70,7 +71,7 @@ private static function sanitize(mixed $value): ?array
         /** @psalm-var array<AdmissionRequirementType> $value */
```

Comment:
Please also add the new value to the `AdmissionRequirementType` psalm type

Reply:
I wonder why Psalm doesn't complain. 🤔

Reply:
Yeah me too, I think it makes sense to remove the psalm type if it's not used somewhere

Reply:
The Psalm definitely requires a type, a new one has been added. Thanks.